ConsoleApp1/Buyer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConsoleApp1
+{
+    internal class Buyer : IBuyer
+    {
+        private string personalInfo;
+        private string name;
+        private List<IReceipt> receipts = new List<IReceipt>();
+
+        public Buyer(string name, string personalInfo)
+        {
+            this.name = name;
+            this.personalInfo = personalInfo;
+        }
+        public string PersonalInfo { get { return personalInfo; } }
```

Comment:
if you use only getter it's better to use `public string PersonalInfo => personalInfo;`
Make changes for all getters in the project